<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSettingsObject</class>
 <widget class="QWidget" name="TabSettingsObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>775</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinThreshold3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> eV</string>
     </property>
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinThreshold">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> eV</string>
     </property>
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDynamicRange">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Dynamic Range:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>190</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboDynamicRange">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>1.67772e+07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>65535</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>255</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="btnSetThreshold">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>20</red>
           <green>20</green>
           <blue>20</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>20</red>
           <green>20</green>
           <blue>20</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>20</red>
           <green>20</green>
           <blue>20</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
     <property name="icon">
      <iconset resource="../include/icons.qrc">
       <normaloff>:/icons/images/rightArrow.png</normaloff>:/icons/images/rightArrow.png</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Settings:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Settings of the detector. 
 #settings#</string>
     </property>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fast</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dynamic Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Low Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Very High Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dynamic HG0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fix Gain 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fix Gain 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force Switch G1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Force Switch G2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Very Low Gain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G1_HG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G1_LG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G2_HC_HG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G2_HC_LG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G2_LC_HG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G2_LC_LG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G4_HG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G4_LG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Undefined</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uninitialized</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblThreshold">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinThreshold2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> eV</string>
     </property>
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboSettings</tabstop>
  <tabstop>spinThreshold</tabstop>
  <tabstop>spinThreshold2</tabstop>
  <tabstop>spinThreshold3</tabstop>
  <tabstop>btnSetThreshold</tabstop>
  <tabstop>comboDynamicRange</tabstop>
 </tabstops>
 <resources>
  <include location="../include/icons.qrc"/>
 </resources>
 <connections/>
</ui>
